app/wizards/placements/add_placement_wizard/preview_placement_step.rb

```diff
@@ -0,0 +1,5 @@
+class Placements::AddPlacementWizard::PreviewPlacementStep < Placements::BaseStep
+  delegate :build_placement, to: :wizard
+
+  alias_method :placement, :build_placement
```

Comment:
In a parallel universe, this would be written as:

```suggestion
  def placement
    @placement ||= wizard.build_placement
  end
```

Somehow the use of `delegate` paired with `alias_method` feels like it carries more cognitive load and requires more interpretation to follow.

But... _tomayto, tomahto_ 🍅 it's all the same in the end.

Reply:
Yeah I know what you mean with that, I think I'll use the method if you think it's better. I _need_ it to be `placement` which is a shame for the cleanliness.